packages/ios-client/scripts/lint_check.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+
+echo "Running swift-format:check for ios files \n"
+swift-format format -i -r ./ --configuration swift-format-config.json
+swift-format lint -s -r ./ --configuration swift-format-config.json
+
```

Comment:
**question:** What's the purpose of this being the same as lint.sh? 🤔

Reply:
I think I just re-used convention. But it makes sense to remove duplicated file